R/nest.R

```diff
@@ -106,39 +156,52 @@ nest <- function(.data, ..., .names_sep = NULL, .key = deprecated()) {
       i = "Did you want `{(.key)} = {cols_fixed_label}`?"
     ))
 
-    return(nest(.data, !!!cols))
+    return(nest(.data, !!!cols, .by = {{ .by }}))
```

Comment:
Supports `df %>% nest(y, .by = x)` where we would get the warning about forgetting `data = y` but then it would continue as expected